File: modulo1_C/exercicios/mario1.c
#include<stdio.h>
#include<cs50.h>

int times(void);


int main(void)
{
 int a=times();

 for(int t=1;t<=a;t++){
     for(int e=a-t;e>=1;e--){
         printf(" ");
     }
     for(int b=1;b<=t;b++){
         printf("#");
     }
      printf("\n");
 }

}


int times(void){
  int n;

  do
  {
       n=get_int("height:");
  }
  while(n<=0 || n>=9);
  /*
  if(n<1)
  {
      printf("false");
  }
  else if(n>8)
  {
       printf("false");
  }
  else
  {
     printf("passou");
  }
*/
 return n;

}

File: modulo6_python/exercicios/mario1.py
import cs50

def main():
   height=0
   while True:
        n = cs50.get_int("Height: ")
        if n > 0 and n<9:
            height+=n
            break

   for i in range(height):
       for t in range(height-i):
           print(" ",end="")

       for j in range(i):
           print("#",end="")

       print("\n")


main()